fetch.bs

```diff
@@ -1170,45 +1170,56 @@ or "<code>worker</code>".
 
 <hr>
 
-<p>To <dfn export for=request id=concept-request-transmit-body>transmit body</dfn> for a
-<a for=/>request</a> <var>request</var>, run these steps:
+<p>To <dfn export for=request id=concept-request-transmit-body-chunk>transmit a chunk</dfn> for a
+promise <var>p</var>, run these steps <a>in parallel</a>:
```

Comment:
It seems you still need to pass request or something as an argument here, since otherwise what do you end up doing the recursion on? _body_ below appears out of nowhere.